Headers/AppKit/NSCell.h

```diff
@@ -123,6 +123,16 @@ enum {
 };
 typedef NSUInteger NSCellStateValue;
 
+/*
+ * Control state values as of 10.13
+ */ 
+enum { 
+  NSControlStateValueMixed	= -1,
+  NSControlStateValueOff	=  0,
+  NSControlStateValueOn		=  1
```

Comment:
Indentation of the "=" looks wrong.

Reply:
Weird, I don't know what is wrong with my emacs settings... Ugh.

Reply:
Removed the extra spaces.